test/environment_test.rb

```diff
@@ -23,17 +23,19 @@
         Dir.chdir($pwd)
       end
 
-      it 'sets env vars from .env' do
-        ENV['FOO'].must_equal 'bar'
-      end
+      describe 'ignores .env' do
+        it 'does not sets env vars from .env' do
+          ENV['FOO'].must_be_nil
+        end
 
-      it 'sets port from .env' do
-        @env.port.must_equal 42
-      end
+        it 'does not sets port from .env' do
```

Comment:
@aderyabin The fact that we're ignoring `.env` doesn't mean we want to ignore if a port is specified there. Can you please restore this test and maybe moving that port value into another `.env.*` fixture? Thanks!